apps/src/lib/config/genesis.rs

```diff
@@ -305,6 +305,9 @@ pub mod genesis_config {
         // light client attack.
         // XXX: u64 doesn't work with toml-rs!
         pub light_client_attack_min_slash_rate: Decimal,
+        /// Number of epochs above and below (separately) the current epoch to
+        /// consider when doing cubic slashing
+        pub cubic_slashing_window_length: u64,
```

Comment:
Does it have to be symmetric?

Reply:
Not completely necessary, but I don't see a reason why it should be asymmetric. This seems most logical.